fe/fe-core/src/main/java/com/starrocks/catalog/system/information/PipeFileSystemTable.java

```diff
@@ -34,7 +34,8 @@ public static SystemTable create() {
 
                         .column("FILE_NAME", ScalarType.createVarcharType(SystemTable.NAME_CHAR_LEN))
                         .column("FILE_VERSION", ScalarType.createVarcharType(SystemTable.NAME_CHAR_LEN))
-                        .column("FILE_ROWS", ScalarType.BIGINT)
+                        // TODO
```

Comment:
It's better to describe the error example in the commit message.

Reply:
done